.github/workflows/operator-ci.yml

```diff
@@ -66,13 +82,143 @@ jobs:
+          driver: docker
+          start args: '--addons=ingress'
+      - name: Build the Keycloak Docker image
+        working-directory: quarkus/container
+        run: |
+          cp ../dist/target/keycloak-*.tar.gz ./
+          eval $(minikube -p minikube docker-env)
+          docker build --build-arg KEYCLOAK_DIST=$(ls keycloak-*.tar.gz) . -t keycloak:${GITHUB_SHA}
+
       - name: Test operator running in cluster
         working-directory: operator
         run: |
           eval $(minikube -p minikube docker-env)
           mvn clean verify \
-              -Dquarkus.container-image.build=true -Dquarkus.container-image.tag=test \
+              -Dquarkus.container-image.build=true \
               -Dquarkus.kubernetes.deployment-target=kubernetes \
               -Dquarkus.jib.jvm-arguments="-Djava.util.logging.manager=org.jboss.logmanager.LogManager","-Doperator.keycloak.image=keycloak:${GITHUB_SHA}","-Doperator.keycloak.image-pull-policy=Never" \
               --no-transfer-progress -Dtest.operator.deployment=remote \
               -Dtest.operator.kubernetes.ip=$(minikube ip)
+
+  test-olm:
+    name: Test OLM installation
+    runs-on: ubuntu-latest
+    needs: [build]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Update maven settings
+        run: mkdir -p ~/.m2 ; cp .github/settings.xml ~/.m2/
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: ${{ env.JDK_VERSION }}
+          cache: 'maven'
+      - name: Download built keycloak distribution
+        uses: actions/download-artifact@v3
+        with:
+          name: keycloak-distribution
+          path: quarkus/dist/target/
+      - name: Setup Minikube-Kubernetes
+        uses: manusa/actions-setup-minikube@v2.4.3
+        with:
+          minikube version: v1.24.0
+          kubernetes version: v1.22.3
+          github token: ${{ secrets.GITHUB_TOKEN }}
+          driver: docker
+          start args: '--addons=ingress'
+      - name: Build the Keycloak Docker image
+        working-directory: quarkus/container
+        run: |
+          cp ../dist/target/keycloak-*.tar.gz ./
+          eval $(minikube -p minikube docker-env)
+          docker build --label "quay.expires-after=20h" --build-arg KEYCLOAK_DIST=$(ls keycloak-*.tar.gz) . -t keycloak:${GITHUB_SHA}
+
+      - name: Build the operator image
+        working-directory: operator
+        run: |
+          eval $(minikube -p minikube docker-env)
+          mvn clean package \
+              -Dquarkus.container-image.build=true \
+              -Dquarkus.container-image.labels."\"quay.expires-after\""="20h" \
+              -DskipTests
+
+      - name: Install the operator CRDs
+        working-directory: operator
+        run: |
+          kubectl apply -f target/kubernetes/keycloaks.keycloak.org-v1.yml
+          kubectl apply -f target/kubernetes/keycloakrealmimports.keycloak.org-v1.yml
```

Comment:
Why are we installing CRDs? It needs to be done by OLM. If not, then we're doing something wrong. 🙂